### PR TITLE
Added more notes about contribution

### PR DESCRIPTION
**Pull request by RichardLitt**
Trying to encourage participation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,15 +22,24 @@ Similar to the [Rust community's](https://this-week-in-rust.org/), we want to ha
 - Tell active contributors when new versions or tools are available.
 - Attribute developers who have contributed to open-source software related to IPFS.
 
+## Contribute
+
+The Weekly takes a _lot of work_. This work can best be done by being distributed. If you know of anything cool that happens in a given week, anything that other people might be excited about, add it to the weekly by adding a comment about it to the PR or Issue for that week's weekly.
```
Comment (hackergrrl) [inline README.md:27]: I wish we could create a perma link that linked to the latest, to reduce friction to submitting content as much as possible.

File: README.md [hunk 2]
```diff
@@ -22,15 +22,24 @@ Similar to the [Rust community's](https://this-week-in-rust.org/), we want to ha
...
+
+Here are the steps to do that:
+ - Do something awesome, or see something cool.
```
Comment (whyrusleeping) [inline README.md]: I'd swap order here, emphasise that its totally cool to just report something someone else did to the weekly

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md:27]: Any ideas how we can do that?

File: README.md (hunk 2 above)
Comment (RichardLitt) [inline README.md]: Done.

**Comment by RichardLitt** [comment]
@noffle How does this look, otherwise?

**Comment by RichardLitt** [comment]
Got an RFM from @noffle on IRC

**Pull request merged**